conference/templatetags/bootstrap_forms.py

```diff
@@ -0,0 +1,211 @@
+from django import forms
+from django.template import Context, Library, Template
+from django.utils.safestring import mark_safe
+
+register = Library()
+
+
+@register.simple_tag
+def render_form(form):
```

Comment:
How come you've decided to create a form rendering module instead of using something like crispy-forms?

Reply:
Looked simple enough at first glance with a bit more flexibility out of the box, however in hindsight it might have been a good idea to go with crispy with some bootstrap templates. That said it probably wasn't more time consuming to write this couple of functions than integrate crispy given that we basically have only simple forms.

Reply:
That's fair enough - the only problem I see is that the complexity of the forms tends to grow and so will the amount of maintenance. Not a blocker blocker though.